### PR TITLE
Added content on Pantheon-extended privacy

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added content on Pantheon-extended privacy including: 
* Pantheon-extended privacy
* How to access private and privacy marker transactions
* How to create and manage privacy groups

### CODE REVIEW

File: docs/Privacy/How-To/Pantheon-Privacy.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+description: Pantheon-extended privacy
+<!--- END of page meta data -->
+
+# Using Pantheon-extended Privacy 
+
+Pantheon provides an extended implementation of privacy. When using Pantheon-extended privacy, a [privacy
```
Comment (bgravenorst) [inline docs/Privacy/How-To/Pantheon-Privacy.md]: Minor suggestion:
Pantheon provides an extended privacy implementation by allowing a privacy group to be created for a set of participants with a unique privacy group ID. The privacy group ID must be specified when sending private transactions.

File: docs/Privacy/How-To/Pantheon-Privacy.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* [`PRIV` API methods](../../Reference/Pantheon-API-Methods.md#priv-methods)
+
+Use [`priv_createPrivacyGroup`](../../Reference/Pantheon-API-Methods.md#priv_createprivacygroup) to 
+create the privacy group containing the recipients of the private transaction. 
+
+Specify `privacyGroupId` when creating the signed transaction passed as an input parameter to [`eea_sendRawTransaction`](../../Reference/Pantheon-API-Methods.md#eea_sendrawtransaction)
+to create an EEA-compliant private transaction. 
+
+!!! note
+    Support for specifying `privacyGroupId` when using eea_sendTransaction with EthSigner will be available in
```
Comment (bgravenorst) [inline docs/Privacy/How-To/Pantheon-Privacy.md]: must eea_sendTransaction be formatted?

File: docs/Privacy/How-To/Pantheon-Privacy.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Privacy/How-To/Pantheon-Privacy.md]: Fixed

File: docs/Privacy/How-To/Pantheon-Privacy.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Privacy/How-To/Pantheon-Privacy.md]: Fixed

**Pull request merged**